test/unit/Command/ConfigurationTest.php

```diff
@@ -6,77 +6,16 @@
 
 use PHPUnit\Framework\TestCase;
```

Comment:
I didn't move this file :man_facepalming: 
Oh well, something for the future me.